### PR TITLE
Fix: Prefer packages to be installed / updated with --prefer-dist

### PR DESCRIPTION
**Pull request by localheinz**
This PR
- [x] uses `--prefer-dist` when installing packages with `composer`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 language: php
 sudo: false
 
+env:
+  global:
+    -  secure: "2DGkxZvwUaAjYu7pTurUsOqgUfkuYdU1i25uPAuTumAt3AAUEg2VZn0dsU8a1NxPmdS/bcYzkoAzaAoINXj5ELpcuUUnHbMjrcEMmCMg6yGPenhS7rSWgeJV+Vh5vsJy5In/PfEExTEL1GGwTz0bEvpPp42Hz0VI17gSJKltn8d+HbIuZeWoj4zfzV8thOKGEdVYb2g8WEJu2AQBovy5vS8hP84C2lQHaxcl7g6Wbqcl7AejRN/HYvoYuTsNya/rJxTfqBIlL7Icj7+KfNyIHnZzPZZaO+pxIwIXTjulc5bclytf7ubIzmpViCC6TOrltuSuLs9xDqn/haBhBIBaN+l3yto3OovQzSu+tmk5qe9a2Hmonk2NDPJHS11swOT3C2DBGlfhuQsATcCa/FEujOK/GBIrSFnDZ6gqq7iHChhmzV0rhR5za+OnLYmvAN4yZ8Eo1isNVrDQH2nKFXQ3Gcgwo8/iZL2n+TDUSpy1zTu35er8f5SoyxhSKgXHNF+m3OHyEvX2UjpDVYIvMKYNZYIqA8/4bB0u9hv/EgjJgxD3G9Avc3f16+k6Zfn7nvEzLG8H8xBC/hTXXClekp3Ib2ZBILppnBd0wZq9kbA4ncJ7DqaWcBVno3ByfQQAlwjwGoKskX8DPmQ3LDEmwbX9v8w+3AGvoU/IxzoXTmpYHOo="
```
Comment (localheinz) [inline .travis.yml]: @WyriHaximus 

I created a GitHub token for public repositories only and encrypted it using

```
$ travis encrypt GH_TOKEN=...
```

Maybe you want to create a token yourself, under your control?

Comment (WyriHaximus) [inline .travis.yml]: It is already set through the travis UI :) 
image

And travis doesn't exposes it to PR's so someone can't just sneak in an echo to the script to get it :). Can you remove that one?

File: .travis.yml [hunk 2]
```diff
@@ -29,7 +33,7 @@ matrix:
 before_script:
   - composer self-update
   - if [ -n "$GH_TOKEN" ]; then composer config github-oauth.github.com ${GH_TOKEN}; fi;
-  - if [ -z "$dependencies" ]; then composer install; fi;
+  - if [ -z "$dependencies" ]; then composer install --prefer-dist; fi;
```
Comment (WyriHaximus) [inline .travis.yml:35]: Looks one, can you add it to the two composer update commands as well? :+1:

**Comment by localheinz** [comment]
@WyriHaximus 

:+1:

**Comment by WyriHaximus** [comment]
Cheers :+1:

**Pull request merged**